atelier-core/src/builder/mod.rs

```diff
@@ -242,6 +245,18 @@ impl ModelBuilder {
             {
                 self.prelude_namespace.make_shape(shape_name)
             } else {
+                // we couldn't find this shape name
```

Comment:
Can you wrap this in a debug_assertion check please

`if cfg!(debug_assertions) {`